docs/README.md

```diff
@@ -137,24 +138,24 @@ shows an example of a proper formatting:
 
 - The tarball generated in the `target` directory can be unpacked using the following command:
 
-    ```sh
```

Comment:
The indentation of the code snippets is intentional. Having it indented this way aligns the code snippets with the bullet points within MkDocs.

Reply:
I think my VScode auto-formatted this. I will revert.